Packs/Armorblox/Scripts/ArmorbloxSendEmail/ArmorbloxSendEmail.py

```diff
@@ -0,0 +1,90 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+# import demistomock as demisto  # noqa: F401
+# from CommonServerPython import *  # noqa: F401
+"""This is an automation script for sending an email whenever a particular incident is encountered.
+This incident is identified by its remediation action. If the remediation action is ALERT or NEEDS REVIEW, an email
+is automatically sent to the desired admin email/group.
+"""
+
+from typing import Dict, Any
+import traceback
+import smtplib
+
+''' STANDALONE FUNCTION '''
+
+
+# TODO: REMOVE the following dummy function:
+def send_email(email: str, incident_id: str, remediation_action: str) -> str:
```

Comment:
Please: 

1. Use one of the integrations that implement `send-mail` ([MailSender](https://xsoar.pan.dev/docs/reference/integrations/mail-sender-new), [Gmail](https://xsoar.pan.dev/docs/reference/integrations/gmail), etc) instead of having a custom script that implements it
2. add a block checking whether the email client integration is enabled before the mail sender (if it isn't, go to the end of the playbook, perhaps raise some error? this is your call)
3. use **playbook inputs** for the destination email instead of command arguments, so users don't have to modify the playbook (but only change the the input in order to use it (to avoid users detaching the playbook and hence missing future updates)

Reply:
Have deleted the Custom Automation Script and used the `send-mail` ( from Mail Sender (New) Pack). Please be informed the the document is also updated with these changes.